Temporary HTML Pages/TempWeek1.html

```diff
@@ -1,70 +1,67 @@
 <!DOCTYPE html>
```

Comment:
Completely ignore this file. All I did was transfer all the handlebars stuff from groupSettings.handlebars just to make sure it has all the general styling on it.
We could delete the Temp HTML pages folder for all I care.